src/ducks/transactions/TransactionsPage.spec.jsx

```diff
@@ -91,25 +93,6 @@ describe('TransactionsPage', () => {
     root = mount(<Wrapper />, { context, childContextTypes })
   }
 
-  it('should not show the top balance on movements page on non mobile', () => {
-    setup()
-    expect(root.find(TransactionsPageBar).length).toBe(0)
-  })
-
-  it('should show the top balance on movements page on mobile', () => {
-    useMobile()
-    setup()
-    expect(root.find(TransactionsPageBar).length).toBe(1)
-  })
-
-  it('should not show the top balance on movements page on mobile if inside category', () => {
-    useMobile()
-    categoryName = 'dailyLife'
-    subcategoryName = 'supermarket'
-    setup()
-    expect(root.find(TransactionsPageBar).length).toBe(0)
-  })
```

Comment:
It seems we are losing those tests ?

Reply:
Oh yeah I forgot to move it after refacto